### PR TITLE
Fix UAP tests for Console

### PR DESCRIPTION
**Pull request by danmosemsft**

### CODE REVIEW

File: src/System.Console/tests/NegativeTesting.cs [hunk 1]
```diff
@@ -9,17 +9,33 @@
 public class NegativeTesting
 {
     [Fact]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]
```
Comment (stephentoub) [inline src/System.Console/tests/NegativeTesting.cs]: Can we add an issue number or comment as to why they're being skipped?

**Comment by danmosemsft** [comment]
@dotnet-bot test Windows x86 Release Build  please (interop hang again tracked elsewhere)

File: src/System.Console/tests/NegativeTesting.cs [hunk 2]
```diff
@@ -9,17 +9,33 @@
 public class NegativeTesting
 {
     [Fact]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)] // In appcontainer, the stream cannot be opened: there is no Console
```
Comment (ViktorHofer) [inline src/System.Console/tests/NegativeTesting.cs]: @danmosemsft With your recent added PlatformDetection property you should now be able to change the condition to `[ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWinRT))]`. That brings two befinits: 
1. If uapaot runs inside an AppContainer the test will be skipped and won't throw
2. If uap runs outside of an AppContainer (for whatever reason) the test will be executed and won't skipped.

File: src/System.Console/tests/NegativeTesting.cs [hunk 3]
```diff
@@ -9,17 +9,33 @@
...
     public static void OpenStandardOutNegativeTests()
     {
         NegativeConsoleOutputTests(Console.OpenStandardOutput());
     }
 
     [Fact]
+    [SkipOnTargetFramework(~TargetFrameworkMonikers.Uap)] // In appcontainer, the stream cannot be opened: there is no Console
```
Comment (ViktorHofer) [inline src/System.Console/tests/NegativeTesting.cs]: Same as above, I would change all the conditions to IsWinRT or IsNotWinRT now. I will stop commenting on others now.

**Pull request merged**